docs/howtos/README.md

```diff
@@ -1,13 +1,13 @@
 # Fleet How-To Guides
 
-The Fleet documentation provides a number of how-to guides that help you get familar with
+The Fleet documentation provides a number of how-to guides that help you get familiar with
 specific Fleet tasks, such as how to use `ClusterResourcePlacement`, a Fleet API, to place
 resources across different clusters.
 
 > Note
 >
 > If you are just getting started with Fleet, it is recommended that you refer to the
-> [Fleet Getting Started Tutorials](../getting-started/README.md) for an overview of Fleet
+> [Fleet Getting Started Guide](../../README.md) for how to create a fleet. an overview of Fleet
```

Comment:
and please refer to the concept read me too

Reply:
a redundant period?